.github/workflows/maven.yml

```diff
@@ -423,13 +423,19 @@ jobs:
       - name: Run benchmark
         # Note: user/pass should match those used in sample configs.
         run: |
-          java -jar benchbase.jar -b ${{matrix.benchmark}} -c config/sqlserver/sample_${{matrix.benchmark}}_config.xml --create=true --load=true --execute=true --json-histograms results/histograms.json
+          java -jar benchbase.jar -b ${{matrix.benchmark}} -c config/sqlserver/sample_${{matrix.benchmark}}_config.xml -im 1000 -mt advanced --create=true --load=true --execute=true --json-histograms results/histograms.json
```

Comment:
Extend to postgres as well?

Reply:
Postgres needs a plugin installed before this works, is that even possible?

Reply:
It should be.  Can you point me at the extension we need?  Maybe include it in some documentation with the PR too?

Reply:
I had added a reference in the README that's in the monitoring folder. This is the name: pg_stat_statements
Let me know if/how enabling it is possible and I'll add the appropriate tests.